uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/modal/modal-document.js

```diff
@@ -249,7 +253,6 @@ define([
           $scope.document.id = result.document.id;
           $scope.document.case_id = result.document.case_id;
           $scope.document.file_count = $scope.files.length + uploader.queue.length;
-          $scope.document.open = $scope.openNew;
```

Comment:
```js
$scope.document.open = $scope.openNew;
```
This says weather we want to open new modal or not. Since we have removed "Save and New" button as per the wireframe, we won't be needing it any more.